components/apimgt/org.wso2.carbon.apimgt.impl/src/test/java/org/wso2/carbon/apimgt/impl/APIProviderImplTest.java

```diff
@@ -2670,13 +2670,14 @@ public void testDeleteAPI() throws RegistryException, UserStoreException, APIMan
                 WorkflowConstants.WF_TYPE_AM_API_STATE)).thenReturn(wfDTO);
         Mockito.doNothing().when(apiStateWFExecutor).cleanUpPendingTask("REF");
 
-        apiProvider.deleteAPI(identifier);
+        apiProvider.deleteAPI(identifier, apiUuid);
     }
 
     @Test(expected = APIManagementException.class)
     public void testDeleteAPI_RegistryException() throws RegistryException, UserStoreException, APIManagementException,
             WorkflowException {
         APIIdentifier identifier = new APIIdentifier("admin-AT-carbon.super", "API1", "1.0.0");
+        String apiUuid = "12345w";
```

Comment:
What's this?

Reply:
It's a sample api_UUID for testing purpose

Reply:
Same as above